music_assistant/server/providers/jellyfin/const.py

```diff
@@ -41,6 +41,7 @@
 
 ITEM_TYPE_ALBUM: Final = "MusicAlbum"
 ITEM_TYPE_ARTIST: Final = "MusicArtist"
+ITEM_TYPE_ARTIST_2: Final = "MusicArtists"

```

Comment:
should the name of this constant not just be ITEM_TYPE_ARTISTS or even ITEM_TYPE_MUSICARTISTS ?

Reply:
I've update it now to ITEM_TYPE_MUSICARTISTS